### ISSUE TITLE
Cannot apply existing template while the page is in edit mode

### ISSUE THREAD

**Issue by valadas** [DNNCommunity/DNN.FormAndList#39]
When the page is in edit mode, if we try to apply an existing template, it just refreshes and the template is not applied. Closing edit mode makes it work.

Dnn 9.2 with F&L 6.5.0

**Comment by SCullman** [comment]
I cannot reproduce in DNN8.4

**Comment by valadas** [comment]
Yeah, I also could not reproduce after that. I _think_ it only happens when you do it right after placing the module on the page, just refreshing seams to solve it.
